packages/cmr-client/CMR.js

```diff
@@ -123,14 +124,14 @@ class CMR {
   }
 
   /**
-   * Return object containing CMR request headers
+   * Return object containing CMR request headers for PUT / POST / DELETE
    *
    * @param {Object} params
    * @param {string} [params.token] - CMR request token
    * @param {string} [params.ummgVersion] - UMMG metadata version string or null if echo10 metadata
    * @returns {Object} CMR headers object
    */
-  getHeaders(params = {}) {
+  getWriteHeaders(params = {}) {
```

Comment:
I don't understand why the write headers don't need the token, but the read headers do.  But I'm sure there's a reason.

Reply:
They do have the token, but github was hiding that line. It's on line 143.